### ISSUE TITLE
Flag aria-checked on an input type=checkbox or radio

### ISSUE THREAD

**Issue by carmacleod** [IBMa/equal-access#347]
Flag aria-checked on an input type=checkbox or radio, as the element's default checked attribute / state always wins out anyway.

https://w3c.github.io/html-aria/tests/checked-test.html

This can either be a warning (telling the author that their aria-checked attribute is useless on input elements that allow the checked attribute) or a violation (telling the author not to do that).

**Comment by MHoov** [comment]
As of June 1
- [x] general rule description (SME & Dev)
- [x] testcases written (Dev)
- [ ] rule developed  (Dev)
- [ ] message written (SME)
- [ ] help written (SME)

**Comment by scottaohara** [comment]
ARIA in HTML (w3c/html-aria#372) is tightening the rule on use of `aria-checked`, particularly due to w3c/aria#1622 and the fact that `aria-checked=mixed` will never be able to be exposed by browsers that correctly follow the ARIA spec.

we would request that this at the very least this be called out as an author error if using aria-checked=mixed on a checkbox.

There is a bug in chromium now where aria-checked incorrectly takes precedent over the native checked / dirty checked state of an element, which can cause inaccurate information to be exposed if the checked state and the aria-checked attribute are out of alignment: https://bugs.chromium.org/p/chromium/issues/detail?id=1260371

**Comment by carmacleod** [comment]
@MHoov Where can I find the "general rule description (SME & Dev)" and testcases that were written as of June 1? I don't know where to look.

**Comment by aliunwala** [comment]
Notes from Triage Meeting:

If aria tag "overrides" the native elements state in an AT then this should be a violation. 

According to a quick chrome test the aria tag overrides the native behavior. So we are expecting this to be a Violation.

**Comment by marcjohlic** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Equal-Access-61ae71f9f8b4ec001e136462/issues/IBMa/equal-access/347?planning-poker) with ZenHub @aliunwala @mbgower @shunguoy @tombrunet @philljenkins @drjoho

**Comment by shunguoy** [comment]
this might be expanded to a more generic case when a user explicitly specifies a role that is the same or conflicts with the implicit role.
